swap_meet/vendor.py

```diff
@@ -1,2 +1,87 @@
+from swap_meet.item import Item
 class Vendor:
-    pass
\ No newline at end of file
+
+    def __init__ (self, inventory=None):
+       self.inventory = inventory
+       if inventory is None:
+           self.inventory = []
+
+
+  
+
+    def add(self, item):
+        self.inventory.append(item)
+        return item
+
+    def remove(self, item):
+        if item not in self.inventory:
+            return False
+        else :
+             self.inventory.remove(item)
+        return item
+
+    def get_by_category(self, category):
+        items = []
+        for item in self.inventory:
+
+            if item.category == category:
+                items.append(item)
+        return items
+
+          
+
+
+    def swap_items(self,vendor, my_item, their_item ):
+        if my_item not in self.inventory or their_item not in vendor.inventory:
+            return False
+        self.inventory.append(their_item)
+        self.inventory.remove(my_item)
+        vendor.inventory.append(my_item)
+        vendor.inventory.remove(their_item)
```

Comment:
you can use the instance methods you already created so `self.add(their_item)` and `self.remove(my_item)`